### DISCUSSION TITLE
Reset one or all habits

### DISCUSSION THREAD

**Discussion by marmonto** [iSoron/uhabits#1016] [Feature Requests]
I would like the option to reset a specific habit to 0

**Comment by hiqua** [comment]
Why not create a new one?

**Reply by ghost** [reply]
> Why not create a new one?

Because resetting would be easier and quicker

**Comment by iSoron** [comment]
This has been suggested in the past. Here are my reasons for not implementing it: https://github.com/iSoron/uhabits/issues/61#issuecomment-199942384

That being said, if you think this feature is important to you, please feel free to upvote it.
